wallet/wallet.go

```diff
@@ -3773,9 +3773,33 @@ func (w *Wallet) TotalReceivedForAddr(addr dcrutil.Address, minConf int32) (dcru
 	return amount, nil
 }
 
-// SendOutputs creates and sends payment transactions. It returns the
-// transaction hash upon success
+// CreateTxOptions allows changing the default behavior of how outputs are constructed
+// when calling `SendOutputsWithOptions`
+type CreateTxOptions struct {
+	// Determines whether the fee comes from the change output or from the payment recipient
+	RecipientPaysFee bool
+}
+
+// NewCreateTxOptions constructs an instance of `CreateTxOptions`, which allows
+// configuring how a transaction is built via methods that accept it as a parameter.
+func (w *Wallet) NewCreateTxOptions(recipientPaysFee bool) *CreateTxOptions {
```

Comment:
This function should not be added.  The struct with exported fields lets the caller set any option they want while not specifying features they don't care about, and not needing to know about new fields added in future versions.  This function doesn't share these properties and updating it for a new option would require a major version bump each time.